src/main/java/ToDo.java

```diff
@@ -0,0 +1,19 @@
+/**
+ * Encapsulates the todo task into a class.
+ * The class supports initializing the todo task and
+ * converting the todo task to a string.
+ */
+public class ToDo extends Task {
```

Comment:
Just a suggestion: Maybe can rename them to TodoTask, EventTask and DeadlineTask so that programmers that doesn't know about our Duke project knows that an Event is a type of Task